rpcserver.go

```diff
@@ -1201,6 +1201,46 @@ func (r *rpcServer) ListPeers(ctx context.Context,
 	return resp, nil
 }
 
+func (r *rpcServer) ChangePassword(ctx context.Context,
+	in *lnrpc.ChangePasswordRequest) (*lnrpc.ChangePasswordResponse, error) {
+
+	// Check macaroon to see if this is allowed.
+	if r.authSvc != nil {
+		if err := macaroons.ValidateMacaroon(ctx, "changepassword",
+			r.authSvc); err != nil {
+			return nil, err
+		}
+	}
+
+	// Check if the current password provided is blank. This most likely
+	// means the wallet came from a --noencryptwallet state and the user
+	// does not know of the default password. In that case, we set it to the
+	// default.
+	if len(in.CurrentPassword) == 0 {
+		in.CurrentPassword = defaultWalletPw
```

Comment:
To avoid the breaking change, you can have one private and one public variable, and set them to "private" and "hello" here.